addOns/ascanrulesAlpha/CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Maintenance changes.
 
 ### Fixed
- - Terminology
+ - Terminology.
+ - SocketTimeoutException in the LDAP Injection scan rule.
```

Comment:
I may be missing something, but I can't see a `SocketTimeoutException` being handled in the LDAP Injection scan rule?

Reply:
My bad, didn't realise `sendAndReceive(originalmsg);` was what was causing a `SocketTimeoutException`.